ArcGISRuntimeSDKQt_CppSamples/Scenes/DisplayScenesInTabletopAR/main.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifdef Q_OS_WIN
+#include <Windows.h>
+#endif
+
+#include "DisplayScenesInTabletopAR.h"
+
+#include <QDir>
+#include <QGuiApplication>
+#include <QQmlApplicationEngine>
+#include <QSurfaceFormat>
+
+#include "ArcGISArView.h"
+
+#define STRINGIZE(x) #x
+#define QUOTE(x) STRINGIZE(x)
+
+int main(int argc, char *argv[])
+{
+#if defined(Q_OS_IOS) || defined(Q_OS_ANDROID)
+  qputenv("QSG_RENDER_LOOP", "basic");
+#endif
```

Comment:
does this sample do anything on desktop platforms? what's the expectations? Might be good to add a comment somewhere

Reply:
This lines are necessary on Mobile because there are some thread conflicts in Qt for rendering. On Desktop, rendering loop works fine in threads.